Makefile

```diff
@@ -137,9 +138,21 @@ helphtml: | nvim build/runtime/doc/tags
 functionaltest functionaltest-lua unittest benchmark: | nvim
 	$(BUILD_TOOL) -C build $@
 
-lintlua lintsh lintpy lintuncrustify lintc lintcfull check-single-includes generated-sources lintcommit lint formatc formatlua format: | build/.ran-cmake
+check-single-includes generated-sources formatc formatlua format: | build/.ran-cmake
 	$(CMAKE_PRG) --build build --target $@
 
+build/CTestTestfile.cmake:
```

Comment:
Isn't this basically the same as the rule `build/.ran-cmake`?

Reply:
In a way yes, yet this is more precise and more specifically I needed this so that those target won't trigger if `build/CTestTestfile.cmake` doesn't exist yet, since `build/.ran-cmake` might already exist already.

Reply:
I'm pretty sure this isn't correct. Running

```
make distclean
make lintlua
```

will result in an error because the dependencies aren't built.

Reply:
I completely missed your comment, sorry! 

Github isn't showing this [comment](https://github.com/neovim/neovim/pull/19189#discussion_r1008667555) as a new activity/review at the bottom of the page :disappointed: